trackio/utils.py

```diff
@@ -8,7 +8,16 @@
 import pandas as pd
 from huggingface_hub.constants import HF_HOME
 
-RESERVED_KEYS = ["project", "run", "timestamp", "step", "time", "metrics"]
+RESERVED_KEYS = [
+    "project",
+    "run",
+    "timestamp",
+    "step",
+    "time",
+    "metrics",
+    "data_type",
+    "x_axis",
```

Comment:
why do we need to add these two?

Reply:
They are used as columns in the DataFrame, and get picked up as `object` type in Pandas. This is curious, and I will investigate a bit more, because now that you mention it I think they should be string type.